From e9834ca4366690dfb97f5aade999b2d611d509bc Mon Sep 17 00:00:00 2001
From: Jonathan Bell <jonathan@raspberrypi.com>
Date: Tue, 5 Mar 2024 10:05:16 +0000
Subject: [PATCH 0926/1085] dt-bindings: add additional RP1 PLL output channels

PLL_AUDIO has a ternary divider (a copy of the secondary divider) and
PLL_VIDEO has a primary phased output.

Signed-off-by: Jonathan Bell <jonathan@raspberrypi.com>
---
 include/dt-bindings/clock/rp1.h | 4 ++++
 1 file changed, 4 insertions(+)

--- a/include/dt-bindings/clock/rp1.h
+++ b/include/dt-bindings/clock/rp1.h
@@ -50,3 +50,7 @@
 #define RP1_CLK_DPI			40
 #define RP1_CLK_MIPI0_DPI		41
 #define RP1_CLK_MIPI1_DPI		42
+
+/* Extra PLL output channels - RP1B0 only */
+#define RP1_PLL_VIDEO_PRI_PH		43
+#define RP1_PLL_AUDIO_TERN		44
